### DISCUSSION TITLE
Using misc repair kit to cut logs into planks uses duct tape

### DISCUSSION THREAD

**Discussion by Proxiehunter** [cataclysmbn/Cataclysm-BN#3236] [General]
It should logically only use the wood cutting function of the repair kit not charges of duct tape. After all you're not duct taping the logs together before you cut them up.

**Comment by Coolthulhu** [comment]
I guess it's probably because it has wood cutting quality, so it gets the "implied" action.
It may be hard to fix fully properly, but a quick fix would be to make wood cutting action never consume any charges.
